### PR TITLE
[docs][notebook1] minor rephrasing of snap_thresh

### PR DESCRIPTION
**Pull request by dglaeser**

### CODE REVIEW

File: docs_src/notebooks/fractopo_network_1.ipynb [hunk 1]
```diff
@@ -192,7 +192,7 @@
     "   \n",
     "-  `snap_threshold`\n",
     "\n",
-    "    -  To determine topological relationships between traces the abutments between traces should be snapped to some tolerance. This tolerance can be given here. E.g. when digitizing in QGIS with snapping turned on, the tolerance is probably much lower than even `0.001`. This represents the lowest distance between nodes that will be interpreted by `fractopo`. If you are doing millimetre scale or lower interpretations you might have to lower this value, otherwise `0.001` is probably fine.\n",
+    "    -  To determine topological relationships between traces the abutments between traces should be snapped to some tolerance. This tolerance can be given here (in the same unit used for the traces). As an example, when digitizing in QGIS with snapping turned on, the tolerance is probably much lower than `0.001`. This represents the lowest distance between nodes that will be interpreted by `fractopo`. Thus, this value should be carefully chosen depending on the size of the area of interest.\n",
```
Comment (dglaeser) [inline docs_src/notebooks/fractopo_network_1.ipynb:195]:
```suggestion
    "    -  To determine topological relationships between traces the abutments between traces should be snapped to some tolerance. This tolerance can be given here, in the same unit used for the traces. It represents the smallest distance between nodes for which `fractopo` interprets them as two distinct nodes. This value should be carefully chosen depending on the size of the area of interest. As a reference, when digitizing in QGIS with snapping turned on, the tolerance is probably much lower than `0.001`.\n",
```

**Comment by dglaeser** [comment]
@nialov, when going through one of the notebooks again, I stumbled across the description of `snap_threshold` again. Instead of opening another issue, I tried to phrase what I think I got from this and propose it here as a PR. Feel free to close it, or to take some of these changes into a new PR by yourself. I am also sorry for the branch name, I did this in the GH web interface and I seem to have missed the point where I could've given the branch a name... In case this PR would be used, I don't see the option of squashing commits here, which probably should be done in this case (the second commit is from a comment)

I guess the point here is: my gut feeling is that `snap_threshold` is a pretty critical parameter, and I have the feeling that it should best be seen of as a relative threshold, that is, relative to the size of the analyzed domain (rather than the unit of analysis). That's why I proposed some rephrasing. Let me know what you think!

**Comment by nialov** [comment]
Thanks for the pull request and rewording! The `snap_threshold` is a very technical parameter which I can image is a problem for understanding for newcomers of `fractopo`. Any help in making the explanations clearer are very welcome.

I rebased/squashed this pull request in #42, though it was probably not needed. I will check in that pr some other texts regarding the `snap_threshold` if there are more places to improve (there probably is :()

**Comment by dglaeser** [comment]
> Your commit should still add you as contributor to the repo after https://github.com/nialov/fractopo/pull/42 is merged so hopefully you do not mind.

@nialov, all good, thank you!

**Pull request closed without merging**